src/schema.js

```diff
@@ -146,13 +184,15 @@ const attributes = {
     },
     array_attributes: {
         parents: ['saml'],
+        examples: ['saml'],
```

Comment:
The `saml` block already links to the saml example.

There are several attributes with examples links where there are already examples links in their containing blocks.

Reply:
Not sure what we want here. You think that's too many links?